telegram_bot/quiz_bot.py

```diff
@@ -56,3 +57,7 @@ async def download_json_callback(message):
         @self.bot.callback_query_handler(func=lambda call: call.data == BotCommands.TO_START.value)
         async def to_start_callback(call):
             await handle_to_start(bot=self.bot, call=call)
+
+        @self.bot.callback_query_handler(func=lambda call: call.data == BotCommands.VIEW_STATISTIC.value)
+        async def to_view_statistic_callback(call):
```

Comment:
I think the name of the function "view user achievements" better reflects its task